lib/travis/logs/helpers/pusher.rb

```diff
@@ -18,9 +19,8 @@ def push(payload)
 
         def pusher_channel_name(payload)
           channel = ''
-          channel << 'private-' if secure?
-          channel << "job-#{payload['id']}"
-          channel
+          channel += 'private-' if secure?
+          channel + "job-#{payload['id']}"
```

Comment:
I'm slightly curious if this has any performance effect, since it creates more strings (`+` makes a new string whereas `<<` modifies the string in-place, IIRC), and I think this is called for every log part? It's probably fine, since it's just two extra (small?) String objects, but might be worth paying attention to when deploying.